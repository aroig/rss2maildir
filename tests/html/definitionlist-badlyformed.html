<dl>
    <dt>An item
    <dd>It's definition
    <dt>Another item
    <dd>And it's got a much longer definition because we like to make sure that we've got the test wrapping right don't we.
</dl>
