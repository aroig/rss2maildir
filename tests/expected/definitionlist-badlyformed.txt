An item::
    It's definition

Another item::
    And it's got a much longer definition because we like to make sure
    that we've got the test wrapping right don't we.
